### PR TITLE
177 fpc table commands

### PR DESCRIPTION
**Pull request by lorenzo-gomez-windhover**
Add FPC table commands for tuning.

### CODE REVIEW

File: apps/fpc/cdr/Main.bob [hunk 1]
```diff
@@ -469,7 +469,7 @@ Yamcs.issueCommand(widget, '/cfs/$(CPUID)/apps/fpc/GoAround', {});]]></text>
         <name>Text Update_8</name>
         <pv_name>/cfs/cpd/apps/fpc/FPC_HK_TLM_MID._hold_alt</pv_name>
```
Comment (mbenson1) [inline apps/fpc/cdr/Main.bob:470]: Replace the "cpd" in the pv_name with $(CPUID) so the display is portable.

File: apps/fpc/fsw/src/fpc_app.h [hunk 2]
```diff
@@ -188,16 +182,6 @@ class FPC
     int8 m_Pos_Reset_Counter{0};				/* captures the number of times the estimator has reset the horizontal position*/
```
Comment (mbenson1) [inline apps/fpc/fsw/src/fpc_app.h:182]: The "{}" initializers will not be set correctly if the application is reloaded.  Move all these initializers to a function that gets called immediately at startup.

File: apps/fpc/fsw/src/fpc_msg.h [hunk 3]
```diff
@@ -138,6 +198,42 @@ typedef struct
...
+/**
+**  \brief Add Message Flow command.
+*/
+typedef struct
+{
+    /** \brief TC message headers */
+    uint8          ucCmdHeader[CFE_SB_CMD_HDR_SIZE];
+
+    /** \brief Index of the channel for which this command applies */
+    boolean        param;
```
Comment (mbenson1) [inline apps/fpc/fsw/src/fpc_msg.h:233]: Use "osalbool".  Unfortunately, there is no platform independent boolean data type.  Use osalbool so we can ensure that everybody uses the same base type.

File: apps/fpc/fsw/src/fpc_app.h (hunk 2 above)
Comment (lorenzo-gomez-windhover) [reply apps/fpc/fsw/src/fpc_app.h:182]: #185

File: apps/fpc/cdr/Main.bob (hunk 1 above)
Comment (lorenzo-gomez-windhover) [reply apps/fpc/cdr/Main.bob:470]: #185

File: apps/fpc/fsw/src/fpc_msg.h (hunk 3 above)
Comment (lorenzo-gomez-windhover) [reply apps/fpc/fsw/src/fpc_msg.h:233]: #185

**Pull request merged**